package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Why is this in the root of the repository?

We'll have a backend, and some other Rust tools in here. I find it annoying to pollute the root with files like this.

Wouldn't it make more sense to put all the JS stuff under some frontend directory.